notebooks/en/llm_gateway_pii_detection.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>I would place the paragraphs describing <code>reponse</code> under the output of <code>reponse</code>, and the paragraphs describing database record under <code>db_record</code> so it is easier to follow.</p><p> </p><p>"The postgres database stores the chat history..."</p><p> </p><p>For "notes", you can use the nice Tip callout :)</p><p> </p><p>"&gt; [!TIP]</p><p>  &gt; The generate endpoint is actually deprecated for Cohere..."  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Addressed (I think, the TIP and NOTE callouts weren't rendering on my local machine but I trusted the process anyways)</p>

Reply:
<p>It doesn't look like it rendered properly, might you know why that is? :S</p>

Reply:
<p>Oops sorry it wasn't clear. You need to include  <code>&gt;</code> in front of the <code>[!TIP]</code> and text  </p>

Reply:
<p>I have no idea what I am doing wrong with the markdown but I've changed it to something somewhat clearer, if you're cool with it then I'm cool with it</p>

Reply:
<p>You can just copy/paste the below and it'll work :)</p><p> </p><p>&gt; [!TIP]</p><p>&gt; The generate endpoint is actually deprecated for Cohere, so it would be a phenomenal open-source contribution to create and commit an integration for the new Chat endpoint for Cohere's API.</p>

Reply:
<p>For some reason it keeps rendering as seen above :S </p>

Reply:
If you check out the page preview [here](https://moon-ci-docs.huggingface.co/docs/cookbook/pr_109/en/llm_gateway_pii_detection), you'll see that it worked! The only thing is that `[!NOTE]` is not supported, so you'll have to change it to `[!TIP]` instead.